mlflow/server/js/src/experiment-tracking/components/HomeView.tsx

```diff
@@ -63,8 +64,8 @@ class HomeView extends Component<HomeViewProps> {
       );
     }
     return (
-      <div css={{ display: 'flex', height: 'calc(100% - 60px)' }}>
-        <div css={{ height: '100%', paddingTop: 24, display: 'flex' }}>
+      <div css={styles.homeContainer}>
```

Comment:
We've eventually stopped using automatic `(theme) => styleObject` function pattern for emotion because `jest` runtime hates it badly and produces super noisy logs. Whenever possible, please just get `theme` object e.g. from hook and use 
```suggestion
      <div css={styles.homeContainer(theme)}>
```

Reply:
ah and for class components, you can use [HoC](https://src.dev.databricks.com/databricks/universe@master/-/blob/design-system/src/design-system/Hooks/useDesignSystemTheme.tsx?L42)